[2024-04-23T21:30:14.828+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-23T21:30:14.849+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.carga manual__2024-04-23T21:29:42.658451+00:00 [queued]>
[2024-04-23T21:30:14.854+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.carga manual__2024-04-23T21:29:42.658451+00:00 [queued]>
[2024-04-23T21:30:14.854+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-23T21:30:14.861+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): carga> on 2024-04-23 21:29:42.658451+00:00
[2024-04-23T21:30:14.869+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'carga', 'manual__2024-04-23T21:29:42.658451+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/air_quality/air_quality_dag.py', '--cfg-path', '/tmp/tmpt8a48g8g']
[2024-04-23T21:30:14.871+0000] {standard_task_runner.py:91} INFO - Job 51: Subtask carga
[2024-04-23T21:30:14.873+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2056) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-23T21:30:14.873+0000] {standard_task_runner.py:63} INFO - Started process 2058 to run task
[2024-04-23T21:30:14.906+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.carga manual__2024-04-23T21:29:42.658451+00:00 [running]> on host 1b9b58bb0dc3
[2024-04-23T21:30:14.970+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='carga' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T21:29:42.658451+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T21:29:42.658451+00:00'
[2024-04-23T21:30:14.971+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-23T21:30:14.980+0000] {redshift.py:69} INFO - cargando los datos a Redshift
[2024-04-23T21:30:18.240+0000] {redshift.py:84} INFO - Datos insertados exitosamente
[2024-04-23T21:30:18.240+0000] {redshift.py:90} INFO - Conexi√≥n cerrada
[2024-04-23T21:30:18.240+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-23T21:30:18.241+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-23T21:30:18.250+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=carga, execution_date=20240423T212942, start_date=20240423T213014, end_date=20240423T213018
[2024-04-23T21:30:18.293+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-23T21:30:18.309+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T21:30:18.312+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
